iroh-net/src/endpoint.rs

```diff
@@ -1245,21 +1240,13 @@ fn proxy_url_from_env() -> Option<Url> {
 
 /// Returns the default relay mode.
 ///
-/// If the `IROH_FORCE_STAGING_RELAYS` environment variable is set to `1`, it will return `RelayMode::Staging`.
+/// If the `IROH_FORCE_STAGING_RELAYS` environment variable is non empty, it will return `RelayMode::Staging`.
 /// Otherwise, it will return `RelayMode::Default`.
 pub fn default_relay_mode() -> RelayMode {
     // Use staging in testing
-    #[cfg(not(any(test, feature = "test-utils")))]
-    let force_staging_relays = match std::env::var(ENV_FORCE_STAGING_RELAYS) {
-        Ok(value) => value == "1",
-        Err(_) => false,
-    };
-    #[cfg(any(test, feature = "test-utils"))]
-    let force_staging_relays = true;
-
-    match force_staging_relays {
-        true => RelayMode::Staging,
-        false => RelayMode::Default,
+    match std::env::var(ENV_FORCE_STAGING_RELAYS) {
+        Ok(value) if !value.is_empty() => RelayMode::Staging,
+        _ => RelayMode::Default,
```

Comment:
Err. I'm a little confused.
Do we want to match on *both* env vars *and* `cfg(test)` now, or not?
Usage now seems somewhat inconsistent.

Perhaps we should just have a single `fn use_staging_infra() -> bool` somewhere instead of the `ENV_FORCE_STAGING_RELAYS` constant.
Then we can make sure our use is consistent.